kubejs/assets/ftbquests/lang/pl_pl.json

```diff
@@ -0,0 +1,130 @@
+    "chapter.1.quest.13.subtitle": "Wybierz swojego rzemieślnika i rozpocznij swoją podróż.",
+    "chapter.1.quest.13.title": "Urządzenie konstrukcyjne",
+    "chapter.1.quest.14.description.1": "Tinkers' Construct (znany również jako Hephaestus dla Fabric) to mod, który pozwala tworzyć wysoko personalizowane narzędzia, umożliwiając tworzenie każdego komponentu indywidualnie z szeroką gamą opcji materiałowych, każdy z różnymi efektami.\n\n\nNarzędzia z początku gry będą wymagać użycia wielu wzorców (Pattern) i kilku specjalnych stołów. Na razie, zgromadź kilka wzorców (Pattern).\n\n\nNagroda za to zadanie dostarczy dodatkowej wiedzy o funkcjonowaniu procesu, a także materiałów narzędziowych, które możesz użyć i ich efektów.",
+    "chapter.1.quest.14.subtitle": "Zaczynamy od podstaw Tinkers.",
+    "chapter.1.quest.15.description.1": "Żelazo i cynk są niezbędne, ale potrzebujesz także miedzianej łopaty Tinkers'! Nie używaj łopaty kamiennej! Są one narzędziem poziomu drewna!",
+    "chapter.1.quest.15.subtitle": "Mamy żelazo w domu:",
+    "chapter.1.quest.15.title": "Uzyskaj te surowce!",
+    "chapter.1.quest.16.description.1": "Będziesz również potrzebował ryżu, zasiej go i zbierz słomę!",
+    "chapter.1.quest.16.subtitle": "Zaczynamy od lekkiej kuchni, prawda?",
+    "chapter.1.quest.16.title": "Rozpocznij karierę szefa kuchni",
+    "chapter.1.quest.17.description.1": "Wiele nowych kultur w Farmer's Delight, czy możesz je wszystkie zautomatyzować?",
+    "chapter.1.quest.17.subtitle": "Mniam ryż",
+    "chapter.1.quest.17.title": "Uzyskaj nowe materiały",
+    "chapter.1.quest.18.description.1": "Wszyscy moi koledzy nienawidzą stylu na początku gry, idźmy szukać podróżnego stroju dla prawdziwego stylu.\n\nZbuduj kompletny zestaw podróżnego ekwipunku i dam ci nagrodę. Będziesz musiał szukać w REI i książkach Tinkers' samemu. To sprawiedliwe, prawda?",
+    "chapter.1.quest.19.description.1": "\n\nLizardLeliel - Gra ważną rolę w postępach zadań w rozdziałach 1-3 i jest odpowiedzialny za dużą część przepisów i automatyzacji, które możesz znaleźć w pakiecie!\n\n\nEthanicuss - Rozwój struktury datapack (astralrails), wkład muzyczny, pomoc w zadaniach, wiele różnych wpisów w trakcie rozwoju (BARDZO).\n\n\nIris - Niesamowity wkład w muzykę pakietu, w tym oficjalną ścieżkę dźwiękową Create: Astral! Odkryj ją na Spotify przed jej implementacją!\n\n\nChoconeosome - Budowa większości budynków, które zobaczysz w AstralRails!",
+    "chapter.1.quest.19.subtitle": "Laky docenia WSZYSTKICH",
+    "chapter.1.quest.19.task.1.title": "Rzeczy do zapamiętania!",
+    "chapter.1.quest.19.title": "Specjalne podziękowania",
+    "chapter.1.quest.2.description.1": "Konstruktor części jest miejscem, w którym bierzesz wzór, bierzesz materiały bazowe i tworzysz indywidualne komponenty narzędzia.",
+    "chapter.1.quest.2.subtitle": "Zbuduj części swojego narzędzia z tym.",
```

Comment:
"za pomocą tego" instead of "z tym", although I think this should be tinkered with more to achieve the correct context.